### ISSUE TITLE
Impossible to Create a Static Client?

### ISSUE THREAD

**Issue by zicklag** [http-rs/surf#150]
I was just trying to create a `lazy_static!` surf client so that I could reuse the Client across my app, but I couldn't find out how because Client requires a private type argument:

```rust
// NativeClient is private so this doesn't work!
lazy_static! {
    static ref WEB: Client<NativeClient> = surf::Client::new();
}
```

Is there a way around this?

**Comment by goto-bus-stop** [comment]
With the surf 2.0 alpha, and installing `http-client`, you can do `surf::Client<http_client::native::NativeClient>`. The goal is to make the type parameter unnecessary for a full 2.0 release.

**Comment by zicklag** [comment]
OK, cool. Maybe I'll try out the 2.0 alpha then. :+1:

**Comment by Fishrock123** [comment]
Also related to https://github.com/http-rs/surf/issues/69

**Comment by goto-bus-stop** [comment]
I think we can call this fixed since #193 was merged? Thanks!
